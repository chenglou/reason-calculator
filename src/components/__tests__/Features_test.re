open Jest;
open Expect;

let _ =

describe "Features" (fun _ => {
  test "renders" (fun _ => {
    let tree = Features.createElement children::[] () |> ReactShallowRenderer.renderWithRenderer;

    expect tree |> toMatchSnapshot;
  });
});
