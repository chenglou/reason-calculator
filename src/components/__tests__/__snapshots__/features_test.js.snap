// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Features renders 1`] = `
<div
  className={
    Object {
      "data-css-186mv4h": "",
    }
  }
>
  <Feature
    reasonProps={
      Array [
        0,
        "Simple operations",
      ]
    }
  />
  <Feature
    reasonProps={
      Array [
        0,
        "Decimals",
      ]
    }
  />
  <Feature
    reasonProps={
      Array [
        0,
        "Percentage",
      ]
    }
  />
  <Feature
    reasonProps={
      Array [
        0,
        "Positive/Negative",
      ]
    }
  />
  <Feature
    reasonProps={
      Array [
        1,
        "Advanced options",
      ]
    }
  />
  <Feature
    reasonProps={
      Array [
        1,
        "Operation history",
      ]
    }
  />
</div>
`;
